lib/clean/convertToProperType.js

```diff
@@ -43,6 +43,19 @@ function convertToProperType(value, type) {
     if (typeof value === 'number') return new Date(value);
   }
 
+  // Convert to Boolean type
+  if (type === Boolean) {
+    if (typeof value === 'string') {
+      // Convert exact string 'true' and 'false' to true and false respectively
+      if (value === 'true') return true;
+      else if (value === 'false') return false;
```

Comment:
Please change both of these lines to `value.toLowerCase()` and add a test for "FALSE" and "TRUE"